python/lib/sift_py/ingestion/_internal/ingest.py

```diff
@@ -58,13 +59,17 @@ def __init__(
     ):
         ingestion_config = self.__class__._get_or_create_ingestion_config(channel, config)
 
-        if not overwrite_rules:
-            self.__class__._validate_rules_synchronized(
-                channel, ingestion_config.asset_id, config.rules
+        if overwrite_rules:
+            warn(
```

Comment:
I wasn't sure if I was in the clear to just remove this arg, so I did this to be cautious. Happy to fully pull it out if we think we're in the clear for that though.

Reply:
Could we specify the exact version we plan to remove it? Next the upcoming but the one after that and create it a ticket for it?